dist/stacktrace-gps.js

```diff
@@ -38,6 +38,25 @@
 
     }
 
+    /**
+     * Convert a Base64-encoded string into its original representation.
+     * Used for inline sourcemaps.
+     *
+     * @param b64str [String]
+     * @return The original representation of the base64-encoded string.
+     */
+    function _atob(b64str) {
+
```

Comment:
Stupid nitpick: could you consider removing the blank lines at the beginning and end of functions. This is consistent with the rest of the codebase.